#!/usr/bin/env python
PACKAGE = "simple_node"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("takeoff", bool_t,   0, "A Boolean parameter",  False)
gen.add("land", bool_t, 0, "A Boolean parameter", False)
gen.add("toggle_cam", bool_t, 0, "A Boolean parameter", False)
gen.add("cont_switch", bool_t, 0, "A Boolean parameter", False)


exit(gen.generate(PACKAGE, "simple_node", "dynparams"))
